rpcs3/rpcs3qt/msg_dialog_frame.cpp

```diff
@@ -25,7 +25,7 @@ void msg_dialog_frame::Create(const std::string& msg)
 	m_dialog = new custom_dialog(type.disable_cancel);
 	m_dialog->setWindowTitle(type.se_normal ? "Normal dialog" : "Error dialog");
 	m_dialog->setWindowFlags(m_dialog->windowFlags() & ~Qt::WindowContextHelpButtonHint);
-	m_dialog->setWindowOpacity(type.bg_invisible ? 1.0 : 192.0 / 255.0);
+	m_dialog->setWindowOpacity(type.bg_invisible ? 192.0 / 255.0 : 1.0);
```

Comment:
This was mentioned in the initial commit message.
> Bug fix in msg_dialog_frame to set the correct background state of message dialogs (visible or transparent)

Bit 2 of the dialog type corresponds to the background style; a background style of CELL_MSGDIALOG_TYPE_BG_VISIBLE should display a solid background in the dialog window; 
CELL_MSGDIALOG_TYPE_BG_INVISIBLE should display no background (transparent).